go.mod

```diff
@@ -85,7 +85,7 @@ require (
 	github.com/ipfs/go-ipfs-exchange-interface v0.0.1
 	github.com/ipfs/go-ipfs-exchange-offline v0.0.1
 	github.com/ipfs/go-ipfs-files v0.0.8
-	github.com/ipfs/go-ipfs-http-client v0.0.5
+	github.com/ipfs/go-ipfs-http-client v0.1.0-update-libp2p
```

Comment:
Can you make a PR for this? We might as well update go-ipfs.

Reply:
@Stebalien Master in `github.com/ipfs/go-ipfs-http-client` already has the latest mutliaddr stuff but upgrading to it breaks a whole host of other ipfs related stuff in Lotus and I don't wanna do that upgrade as part of this PR.

Reply:
Can we tag v0.0.6 then?

Reply:
@Stebalien  This is done. Please can you approve the PR ?